SPECS/patterns-ceph-containers/patterns-ceph-containers.spec

```diff
@@ -0,0 +1,94 @@
+Summary:        Patterns for the Ceph containers
+Name:           patterns-ceph-containers
+Version:        1.0
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Metapackages
+URL:            http://en.opensuse.org/Patterns
+ExclusiveArch:  x86_64 aarch64 ppc64le s390x
+
+%description
+This is an internal package that is used to create the patterns as part
+of the installation source setup. Installation of this package does
+not make sense.
+
+%package ceph_base
+Summary:        Ceph base
+Group:          Metapackages
+Provides:       pattern() = ceph_base
+Provides:       pattern-category() = Containers
+Provides:       pattern-order() = 3000
+Provides:       pattern-visible()
+Requires:       ca-certificates
+Requires:       ceph
+Requires:       ceph-base
+Requires:       ceph-common
+Requires:       ceph-fuse
+Requires:       ceph-grafana-dashboards
+Requires:       ceph-mds
+Requires:       ceph-mgr
+Requires:       ceph-mgr-cephadm
+Requires:       ceph-mgr-dashboard
+Requires:       ceph-mgr-rook
+#Package currently not supported in mariner, keeping dependency for future reference.
+#Requires:      ceph-mgr-diskprediction-local
+Requires:       ceph-mon
+Requires:       ceph-osd
+Requires:       ceph-prometheus-alerts
+Requires:       ceph-radosgw
+Requires:       cephadm
+Requires:       e2fsprogs
+Requires:       gptfdisk
+Requires:       kmod
+Requires:       lvm2
+#Package currently not supported in mariner, keeping dependency for future reference.
+#Requires: ceph-iscsi
+Requires:       rbd-mirror
+Requires:       rbd-nbd
+
+%description ceph_base
+This provides the base for the Ceph, Rook, Ceph CSI driver packages and containers.
+
+%prep
+# empty on purpose
+
+%build
+# empty on purpose
+
+%files ceph_base
+
+%changelog
+* Mon Oct 04 2021 Max Brodeur-Urbas <maxbr@microsoft.com> - 1.0-1
+- Initial CBL-Mariner import from openSUSE Tumbleweed (license: same as "License" tag).
+- License Verified
+- Modified installation path
```

Comment:
Nit-pick/non-blocker: I think this comment is no longer valid, since there are no paths to talk about.